app.py

```diff
@@ -63,6 +63,9 @@ def stats(api, memory_used_gauge, memory_total_gauge, network_up_gauge, network_
 
     for volume_id in api.storage.volumes_ids:
         volume_status = str(api.storage.volume_status(volume_id))
+        if volume_status not in volume_states:
+            volume_status = "error"
```

Comment:
Wouldn't it be better to raise an error here and log if there are any errors? Maybe that's not the role of a prometheus exporter...

Reply:
Yes maybe logging the value so if there are more states we can patch it